plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -1930,6 +1955,19 @@ public void setTableProperties(ConnectorSession session, ConnectorTableHandle ta
         }
     }
 
+    private MetadataEntry buildMetadataEntry(MetadataEntry metadataEntry, Map<String, String> configuration, long createdTime)
```

Comment:
is `createdTime` time of table creation or time of metadata entry creation?

(in the latter case, the code here could take current time itself)

Reply:
It is set in both the `MetadataEntry` and `CommitInfoEntry`, we're doing this so the two have the same time.